### PR TITLE
Public upstream

### PR DESCRIPTION
**Pull request by bhastings1019**
chore: merge work to update proper upstream. 

This work consists of releasing: 
* PE-146: returns incorrect JSON object for device info
* PE-108: Allow for editing of a player's locally configured name
* PE-74: User should be able to choose level of local logging
* PE-98: Write better documentation for the npm docs page
* PE-109: Add specific usage examples to the help screen of each command

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@brightsign/bsc",
-  "version": "1.0.1",
+  "name": "bsc",
```
Comment (jagreehal) [inline package.json:2]: I'd leave this to match current npm package name

Comment (bhastings1019) [reply package.json:2]: Good catch - that was not intentional.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,107 @@
 # BrightSign CLI API Tool
+<p align="center">
+    <img src="./readmeMedia/PoweredByPurple.jpg" alt="Powered by Purple!" width="400" height="194">
+</p>
 The purpose of this package is to allow users to communicate with the BrightSign's Local DWS (Diagnostic Web Server) REST HTTP APIs through a simple CLI tool, `bsc`.
```
Comment (jagreehal) [inline README.md:6]: maybe make this the link e.g. [Local DWS (Diagnostic Web Server) REST HTTP APIs](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/1172734089/Local+DWS+APIs)

**Review by jagreehal — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bhastings1019) [reply README.md:6]: Updated - and updated the image to link to the same site for now.

**Review by lherlein — APPROVED** [review]

**Pull request merged**